src/components/chat.js

```diff
@@ -854,7 +854,7 @@ class Chat extends Emitter {
                             if (response.data.found) {
                                 this.meta = response.data.chat.meta;
                             } else {
-                                this.update(this.meta);
+                                this.meta = this.meta || {};
```

Comment:
Not exactly what has been requested in change request. 

We need to limit call of `update` function only, if there is any data in `this.meta` (to not make any extra network calls which won't do anything useful). Expected something like this:
```js
} else if (Object.keys(this.meta)) {
    this.update(this.meta);
}
```

Or some other way to ensure, what we can provide any useful meta object.